### PR TITLE
feat: add message to inform user about temporary pack creation

### PR DESCRIPTION
**Pull request by Steffy29**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #UXCT-631
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~Breaking change is mentioned in relevant commits~

## Description

Add message into summary offer change page to inform user about temporary pack creation for change offer and move offer.

### CODE REVIEW

**Review by ghyenne — APPROVED** [review]

File: packages/manager/apps/telecom/src/app/telecom/pack/move/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -293,5 +293,6 @@
   "pack_move_ont_shipping_explain": "L'ONT (Optical Network Termination) est un équipement réseau qui permet de raccorder votre modem à la PTO.",
   "pack_move_ont_shipping_contact": "Contact de livraison",
   "pack_move_ont_shipping_confirm": "Confirmer ce contact de livraison",
-  "pack_move_ont_shipping_addresses_error": "Une erreur est survenue lors du chargement de vos contacts. {{error}}"
+  "pack_move_ont_shipping_addresses_error": "Une erreur est survenue lors du chargement de vos contacts. {{error}}",
+  "pack_move_confirm_temporary_pack_creation": "Durant la transition vers votre nouvelle offre, un pack temporaire sera créé pour assurer la continuité du service.<br>Ce pack sera automatiquement supprimé une fois votre commande Fibre finalisée.<br>Aucune action de modification ou de suppression de votre part n'est nécessaire : le passage vers votre accès Fibre se fera de manière entièrement automatisée."
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/pack/move/translations/Messages_fr_FR.json:297]:
```suggestion
  "pack_move_confirm_temporary_pack_creation_part_1": "Durant la transition vers votre nouvelle offre, un pack temporaire sera créé pour assurer la continuité du service.",
"pack_move_confirm_temporary_pack_creation_part_2": "Ce pack sera automatiquement supprimé une fois votre commande Fibre finalisée.",
"pack_move_confirm_temporary_pack_creation_part_3": "Aucune action de modification ou de suppression de votre part n'est nécessaire : le passage vers votre accès Fibre se fera de manière entièrement automatisée."
```

File: packages/manager/apps/telecom/src/app/telecom/pack/move/resume/move-resume.html [hunk 2]
```diff
@@ -333,6 +333,10 @@ <h3 data-translate="pack_move_doing"></h3>
             </div>
         </div>
 
+        <oui-message type="info" data-ng-if="$ctrl.offer.selected.technoChange">
+            <p data-translate="pack_move_confirm_temporary_pack_creation"></p>
+        </oui-message>
+
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/pack/move/resume/move-resume.html:339]:
```suggestion
        <oui-message type="info" data-ng-if="$ctrl.offer.selected.technoChange">
            <p class="mb-0" data-translate="pack_move_confirm_temporary_pack_creation_part_1"></p>
            <p class="mb-0" data-translate="pack_move_confirm_temporary_pack_creation_part_2"></p>
            <p data-translate="pack_move_confirm_temporary_pack_creation_part_3"></p>
        </oui-message>

```

**Review by ghyenne — DISMISSED** [review]

File: packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.html [hunk 3]
```diff
@@ -304,6 +304,24 @@ <h3 data-translate="telecom_pack_migration_doing"></h3>
...
+                data-translate="telecom_pack_migration_confirm_temporary_pack_creation_line_1"
+            ></p>
+            <p
+                class="mb-0"
+                data-translate="telecom_pack_migration_confirm_temporary_pack_creation_line_2"
+            ></p>
+            <p
+                class="mb-0"
+                data-translate="telecom_pack_migration_confirm_temporary_pack_creation_line_3"
+            ></p>
```
Comment (antonymarion) [inline packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.html:322]: **nitpick**: use ng repeat with [line1, line2, line3] constants defined in constant file

Comment (Steffy29) [reply packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.html:322]: Are you kidding? Use a ng-repeat just to display 3 information lines for the customer, for me you want to use a bazooka to kill a fly.

Comment (antonymarion) [reply packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.html]: Nope, just wanted to compact code and therefore ease code maintenance for the future

Comment (antonymarion) [reply packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.html]: Nope, just wanted to compact code and therefore ease code maintenance for the future

**Review by antonymarion — APPROVED** [review]

File: packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.html [hunk 4]
```diff
@@ -304,6 +304,17 @@ <h3 data-translate="telecom_pack_migration_doing"></h3>
             </div>
         </div>
 
+        <oui-message
+            type="info"
+            data-ng-if="$ctrl.process.selectedOffer.changeTechno"
+        >
+            <p
+                class="mb-0"
+                data-ng-repeat="line in $ctrl.TEMPORARY_PACK_LINES track by $index"
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.html:313]:
```suggestion
                data-ng-repeat="line in [1,2,3] track by $index"
```
If you want reduce the code, in this case i think we can avoid create a constant for just 3 number and directly put the array with 1,2,3

**Review by antonymarion — DISMISSED** [review]

**Review by ghyenne — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Review by antonymarion — APPROVED** [review]

**Pull request merged**